configure.ac

```diff
@@ -1149,6 +1159,14 @@ AC_SUBST(LEVELDB_CPPFLAGS)
 AC_SUBST(LIBLEVELDB)
 AC_SUBST(LIBMEMENV)
 
+dnl SUPPRESSED_CPPFLAGS=SUPPRESS_WARNINGS([$SOME_CPPFLAGS])
+dnl Replace -I with -idirafter in $SOME_CPPFLAGS to suppress warnings from
+dnl headers from its include directories and return the result.
+dnl See -idirafter documentation:
+dnl https://gcc.gnu.org/onlinedocs/gcc/Directory-Options.html
+dnl https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-idirafter-arg
+AC_DEFUN([SUPPRESS_WARNINGS], [`echo $1 |sed -E -e 's/(^| )-I/\1-idirafter /g'`])
```

Comment:
nit: [Use $(...) notation instead of legacy backticked \`...\`.](https://github.com/koalaman/shellcheck/wiki/SC2006)

Reply:
Done.